source/getting-started/application-setup-and-structure.md

```diff
@@ -0,0 +1,134 @@
+Ember is designed to minimize the amount of configuration necessary to get an
+application up and running.  To do this, a number of peripheral tasks to
+application development have been streamlined, such as testing, compiling, and
+serving files.  This allows Ember developers to focus on building apps rather
+than building the support structures that make them run.
+
+Ember includes a _command line interface_ called Ember CLI with helpful tools
+that are immediately available.  In the terminal, a quick `ember --help` shows
+the commands we can use.  For more information on a specific command, we can
+type `ember help command-name`.  This shows us the details of how the command is
```

Comment:
Following the way `ember help` formats variables inside command examples, this could be `ember help <command-name>`